eng/Testing.props

```diff
@@ -1,4 +1,15 @@
 <Project>
+  <PropertyGroup>
+    <IncludeTestUtilities>true</IncludeTestUtilities>
+
+    <!-- By default any test can run on all test platforms -->
+    <RunOnGithubActionsWindows>true</RunOnGithubActionsWindows>
+    <RunOnGithubActionsLinux>true</RunOnGithubActionsLinux>
+    <RunOnAzdoCI>true</RunOnAzdoCI>
```

Comment:
Does this need to be split into two too?

Reply:
_Possibly_, but I haven't seen a use for it in the existing implementations. Currently, we run all tests on the same build agent, so separating the property in to `RunOnAzdoWindows` and `RunOnAzdoLinux` wouldn't add any value.
However, once we move off Helix and (if) start running tests in AzDO on different OS, then adding new properties may make total sense.

Reply:
IMHO, that seems at odds with the reasoning for splitting the properties at all - clarity, and consistency.

Reply:
Agree to a degree. GHA and Helix run on different OS, whereas CI doesn't. So, splitting the property doesn't feel justified.
My primary objective right now is GHA-related test infra. I haven't looked at the Helix/CI in details, and I'm trying to make minimally necessary changes to avoid any disruptions there. 
I agree that splitting the Helix properties may look excessive, and they may be removed in the future. For now, they have minimal impact (if any at all).

Reply:
Isn't "CI" the pipeline (non-helix) tests which run on both Linux, and Windows?

Reply:
At this stage this is totally immaterial.